### ISSUE TITLE
Dynamically generating the RaftJournalSystem#RAFT_GROUP_UUID would be more appropriate

### ISSUE THREAD

**Issue by jianghuazhu** [Alluxio/alluxio#17122]
**Summary**
In RaftJournalSystem, RAFT_GROUP_UUID is a fixed value --- 02511d47-d67c-49a3-9011-abb3109a44c1, which is not conducive to scaling distributed systems. Dynamically generating the RaftJournalSystem#RAFT_GROUP_UUID would be more appropriate.

**Urgency**
For distributed systems, it is important to enhance scalability.

**Comment by jianghuazhu** [comment]
// assign @jianghuazhu

**Comment by jianghuazhu** [comment]
Here are some tests:
`
        String uuid = UUID.randomUUID().toString();
        System.out.println("uuid =  " + uuid);
`
Output:
`
uuid =  42494149-d6ff-4d55-aace-38942b51b86d
`
The output looks consistent with '02511d47-d67c-49a3-9011-abb3109a44c1'.

**Comment by jja725** [comment]
@tcrain @jenoudet do you mind take a look? Thanks

**Comment by jenoudet** [comment]
I agree with you @jianghuazhu, this is something that should be fixed.
